api/src/pcapi/routes/public/individual_offers/v1/bookings.py

```diff
@@ -194,6 +194,50 @@ def validate_booking_by_token(token: str) -> None:
     bookings_api.mark_as_used(booking)
 
 
+@blueprint.v1_bookings_blueprint.route("/keep/token/<token>", methods=["PATCH"])
+@spectree_serialize(
+    on_success_status=204,
+    api=blueprint.v1_bookings_schema,
+    tags=[constants.BOOKING_TAG],
+    resp=SpectreeResponse(
+        **(
+            BOOKINGS_API_BASE_CODE_DESCRIPTIONS
+            | {
+                "HTTP_204": (None, "The booking's validation has been cancelled successfully"),
+                "HTTP_403": (
+                    None,
+                    "This booking's validation cannot cancelled because you do not have the proper right or because the token's status is not used",
+                ),
+                "HTTP_410": (None, "This booking cannot be unvalidated"),
```

Comment:
Est-ce que ça n'est pas trop vague ? "Ça n'est pas possible (et je ne te dirai pas pourquoi)". Ça me fait penser à "[computer says no](https://www.youtube.com/watch?v=oXpqyl2FH-M)". :)

En réalité, on sait pourquoi : c'est parce que la réservation est annulée. On pourrait donc l'indiquer : 
```suggestion
                "HTTP_410": (None, "This booking cannot be unvalidated because it is already cancelled"),
```